Auth/Auth.pm

```diff
@@ -859,6 +861,41 @@ sub get_eduPersonUnScopedAffiliation {
 
 # ---------------------------------------------------------------------
 
+=item get_eduPersonEntitlement
+
+Return value object for $ENV{entitlement}
+
+=cut
+
+# ---------------------------------------------------------------------
+sub get_eduPersonEntitlement {
+    my $self = shift;
+    my $C = shift;
+
+    return Auth::Auth::Entitlement->new(
+        $self->has_expected_shibboleth_entityID($C) ? $ENV{entitlement} : ''
```

Comment:
`$self->has_expected_shibboleth_entityID($C)` always returns false; it's expecting a second parameter of the actual entityID. Perhaps instead:

```perl
$self->get_shibboleth_entityID($C) ? $ENV{entitlement} : ''
```
```

Reply:
Updated to avoid extracting the entityid for this test.